tests/deepsparse/v2/test_basic_pipeline.py

```diff
@@ -19,21 +35,21 @@ class AddOneOperator(Operator):
     input_schema = IntSchema
     output_schema = IntSchema
 
-    def run(self, inp: IntSchema, context: Context) -> OperatorSchema:
-        return IntSchema(value=inp.value + 1)
+    def run(self, inp: IntSchema, context: Context) -> Dict:
+        return {"value": inp.value + 1}
 
 
 class AddTwoOperator(Operator):
     input_schema = IntSchema
     output_schema = IntSchema
 
-    def run(self, inp: IntSchema, context: Context) -> OperatorSchema:
-        return IntSchema(value=inp.value + 2)
+    def run(self, inp: IntSchema, context: Context) -> Dict:
+        return {"value": inp.value + 2}
 
 
 AddThreePipeline = Pipeline(
-    stages=[AddOneOperator(), AddTwoOperator()],
-    router=Router(),
+    ops=[AddOneOperator(), AddTwoOperator()],
+    router=LinearRouter(end_route=2),
     schedulers=[OperatorScheduler()],
```

Comment:
might as well throw a simple composability test in here

Reply:
like router vailidity?